### PR TITLE
Add support for Heatmap panels

### PR DESCRIPTION
**Pull request by K-Phoen**
Hi!

In this PR, I add support for Heatmap panels: https://grafana.com/docs/grafana/v6.7/getting-started/intro-histograms/#heatmaps
They're available since v6.7 at least (not sure about prior versions)

### CODE REVIEW

File: panel.go [hunk 1]
```diff
@@ -311,6 +313,51 @@ type (
...
+			Show bool `json:"show"`
+		} `json:"xAxis"`
+		XBucketNumber *float64 `json:"xBucketNumber"`
+		XBucketSize   *float64 `json:"xBucketSize"`
+		YAxis         struct {
+			Decimals    *int     `json:"decimals"`
+			Format      string   `json:"format"`
+			LogBase     int      `json:"logBase"`
+			Show        bool     `json:"show"`
+			Max         *float64 `json:"max"`
```
Comment (GiedriusS) [inline panel.go:353]: I think this is a string. In one of my example dashboards I got:
```
      "yAxis": {
        "decimals": 3,
        "format": "short",
        "logBase": 1,
        "max": "100",
        "min": "0",
        "show": true,
        "splitFactor": null
      },
  ```

File: panel.go [hunk 2]
```diff
@@ -311,6 +313,51 @@ type (
...
+			Min         *float64 `json:"min"`
```
Comment (GiedriusS) [inline panel.go:354]: Ditto.

File: panel.go [hunk 3]
```diff
@@ -311,6 +313,51 @@ type (
...
+		Tooltip         struct {
+			Show          bool `json:"show"`
+			ShowHistogram bool `json:"showHistogram"`
+		} `json:"tooltip"`
+		TooltipDecimals int `json:"tooltipDecimals"`
+		XAxis           struct {
+			Show bool `json:"show"`
+		} `json:"xAxis"`
+		XBucketNumber *float64 `json:"xBucketNumber"`
+		XBucketSize   *float64 `json:"xBucketSize"`
```
Comment (GiedriusS) [inline panel.go:347]: I think this is a string at least on 6.7.1:
```
      "xAxis": {
        "show": true
      },
      "xBucketNumber": 0,
      "xBucketSize": "2",
      "yAxis": {
  ```

File: panel.go [hunk 4]
```diff
@@ -311,6 +313,51 @@ type (
...
+		Cards struct {
+			CardPadding *float64 `json:"cardPadding"`
+			CardRound   *float64 `json:"cardRound"`
+		} `json:"cards"`
+		Color struct {
+			CardColor   string   `json:"cardColor"`
+			ColorScale  string   `json:"colorScale"`
+			ColorScheme string   `json:"colorScheme"`
+			Exponent    float64  `json:"exponent"`
+			Min         *float64 `json:"min,omitempty"`
```
Comment (GiedriusS) [inline panel.go:326]: This is correct :+1:

**Review by GiedriusS — CHANGES_REQUESTED** [review]
LGTM, just a few suggestions on the types in your structs!

**Review by GiedriusS — APPROVED** [review]
:heart:

**Comment by K-Phoen** [comment]
Thanks for the careful review and merge @GiedriusS :muscle:

**Pull request merged**